src/app/demo/all-exercises/all-exercises.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AllExercisesComponent } from './all-exercises.component';
+
+// TODO: Fix this test suite.
```

Comment:
Please fix this and write tests. It looks like component only has 1 dependency so it shouldn't be very difficult, but maybe all of the directives in the template would need to be mocked out.

If this really feels like a big piece of work, then sounds good to me to skip this for now. Some dependencies might be difficult to mock.